cp3pt0-deployment/common/utils.sh

```diff
@@ -256,6 +256,32 @@ function is_sub_exist() {
     is_exist=$(echo "$name" | grep -w "$package_name")
 }
 
+function check_namespace(){
+    local namespace=$1
+    if [[ -z "$(${OC} get namespace ${namespace} --ignore-not-found)" ]]; then
+        error "Namespace ${namespace} does not exist"
+    fi
+}
+
+function check_cert_manager(){
+    csv_count=`$OC get csv |grep "cert-manager"|wc -l`
+    if [[ $csv_count == 0 ]]; then
+        error "Missing a cert-manager"
+    fi
+    if [[ $csv_count > 1 ]]; then
+        error "Multiple cert-manager csv found. Only one should be installed per cluster"
+    fi
+}
+
+function check_licensing(){
+    csv_count=`$OC get csv |grep "ibm-licensing"|wc -l`
```

Comment:
I think `ibm-licensing` operator is installed in its own namespace, and it will NOT extend its CSV into target namespace until OprandRequest with `ibm-licensing-operator` is created.
In other word, we could not get the accurate result by only checking the licensing CSV in specific namespace.

Reply:
The proper way to check whether LicenseService is running would be to check for resource of kind `IBMLicensing` named `instance`

`oc get IBMLicensing | wc -l`